CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# v1.19.0 Release - 01/28/2021
+
+**Linux**
+`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.19.0/skaffold-linux-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+
+**macOS**
+`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.19.0/skaffold-darwin-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+
+**Windows**
+ https://storage.googleapis.com/skaffold/releases/v1.19.0/skaffold-windows-amd64.exe
+
+**Docker image**
+`gcr.io/k8s-skaffold/skaffold:v1.19.0`
+
+From release v1.19.0, skaffold will now collect anonymized Skaffold usage data. 
```

Comment:
We should mention that these are opt-in

Reply:
they are actually opt-out, not opt-in

Reply:
whoops my bad, thanks for the correction

Reply:
done. added the summary. Want users to go to the doc link to know the command and example.